### PR TITLE
Pointers to logging models, ViaQ, OTEL and transformation.

### PR DESCRIPTION
**Pull request by alanconway**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
...
+## Logging
+
+Current logging data model:
+- [ViaQ Data Model](https://viaq.github.io/documentation/data_model/public/data_model.html)
+
+Relevant OTEL models:
+- [Logs Data Model](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md)
+- [Kubernetes Resource Semantic Conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/k8s.md)
+
+Current OTEL support is Tech Preview, JSON over HTTP only.
```
Comment (pavolloffay) [inline README.md:13]: Could we add some pointers to docs how to enable this? Which version supports it?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+
+[Transformation from ViaQ to OTEL](https://github.com/openshift/cluster-logging-operator/blob/master/internal/generator/vector/normalize/schema/otel/transform.go#L16)
+
+``` vrl
```
Comment (pavolloffay) [inline README.md:17]: Why are we copyting the `transform.go` to readme? I would rather see a configuration how to enable it

File: README.md [hunk 3]
```diff
@@ -1,2 +1,49 @@
 # observability-data-model
 Documentation for telemetry data model and semantic conventions
+
+## Logging
```
Comment (pavolloffay) [inline README.md:4]: Could we move this to a seprate file e.g. `logging.md` ?

File: README.md (hunk 1 above)
Comment (alanconway) [reply README.md:13]: Will do.

File: README.md (hunk 3 above)
Comment (alanconway) [reply README.md:4]: Will do

File: logging.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+	.resources.container.name = del(.kubernetes.container_name)
+	.resources.container.id = del(.kubernetes.container_id)
+
+	# split image name and tag into separate fields
+	container_image_slice = split!(.kubernetes.container_image, ":", limit: 2)
+	if null != container_image_slice[0] { .resources.container.image.name = container_image_slice[0] }
+	if null != container_image_slice[1] { .resources.container.image.tag = container_image_slice[1] }
+	del(.kubernetes.container_image)
+
+	# kuberenetes
```
Comment (simonpasquier) [inline logging.md:35]: (nit) I see the typo in the original code too
```suggestion
	# kubernetes
```

**Pull request closed without merging**